src/gt4py/gtc/common.py

```diff
@@ -475,3 +475,24 @@ def _impl(cls, values):
         return values
 
     return root_validator(pre=True, allow_reuse=True)(_impl)
+
+
+class AxisBound(Node):
```

Comment:
Can you update the other IRs to use AxisBound from common as well or add a TODO there for later.